deploy/aws/README.md

```diff
@@ -22,7 +22,12 @@
   The easist way to install `aws-iam-authenticator` is to download the prebuilt binary:
 
   ``` shell
+  # Download binary for Linux
   curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.12.7/2019-03-27/bin/linux/amd64/aws-iam-authenticator
+
+  # Or, download binary for macOS
+  curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.12.7/2019-03-27/bin/darwin/amd64/aws-iam-authenticator
+
```

Comment:
Maybe using the code below is simpler and more general
```
os=$(uname -s| tr '[:upper:]' '[:lower:]')
curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.12.7/2019-03-27/bin/$os/amd64/aws-iam-authenticator
```

Reply:
I don't have any mac for test, if you can confirm this works for macOS?

Reply:
Yes, it works for macOS and Linux

Reply:
@AstroProfundis The [link](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html) you mentioned already has an elegant way to install aws-iam-authenticator.
For macOS, install with homebrew. For linux install with the curl to fetch the binary. It's inappropriate to provide the tricky script in the guide.

Reply:
I don't think homebrew is available to every macOS installation by default...

Reply:
Yes, it's not builtin on macOS. But almost all technical users install that. So I think provide homebrew command is reasonable.

Reply:
This definitely can't pass Queeney's test, the "almost all technical users" is not fitting the target user groups.

Reply:
You can add a note in the document that before users install aws-iam-authenticator with homebrew, they need to make sure that they have installed homebrew first.

Reply:
Then we're introducing new dependency and copying almost all major parts of AWS document, what's the point of avoiding out-linking again?

Reply:
Yeah, you are right. So if the curl way does not require any new dependency for macOS, I suggest we introduce the curl way in our document then. If users prefer the homebrew way, they can still click the link and find the guide in the AWS document.